<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LIN65004458" failures="1" tests="1" name="Default test" time="20.73" errors="0" timestamp="2025-02-10T14:28:34 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="19.945">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at stepDefinitions.OrderStepDefiniton.checkout_submit_order(OrderStepDefiniton.java:54)
at âœ½.Checkout "IPHONE" and submit the order(file:///C:/Users/kakpavan/eclipse-workspace/SeleniumFramework25/src/test/java/cucumber/SubmitOrder.feature:15)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
